Gopkg.toml

```diff
@@ -68,3 +68,7 @@
 [[constraint]]
   branch = "master"
   name = "github.com/giantswarm/versionbundle"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/prometheus/client_golang"
```

Comment:
A simple `dep ensure -update` did not work. I had to add this.

Reply:
Can you share the error output or it's gone?

Reply:
```
$ dep ensure -update
Solving failure: No versions of github.com/giantswarm/microkit met constraints:
	master: Could not introduce github.com/giantswarm/microkit@master, as it has a dependency on github.com/prometheus/client_golang with constraint master, which does not allow the currently selected version of 0.2.0
```

Reply:
```
$ dep status
PROJECT                                           CONSTRAINT        VERSION           REVISION  LATEST   PKGS USED
github.com/PuerkitoBio/purell                     v1.1.0            v1.1.0            0bcb03f   0bcb03f  1
github.com/PuerkitoBio/urlesc                     branch master     branch master     de5bf2a   de5bf2a  1
github.com/beorn7/perks                           branch master     branch master     4c0e845   4c0e845  1
github.com/cenkalti/backoff                       v1.1.0            v1.1.0            61153c7   61153c7  1
github.com/coreos/go-semver                       v0.2.0            v0.2.0            8ab6407   8ab6407  1
github.com/davecgh/go-spew                        v1.1.0            v1.1.0            346938d   346938d  1
github.com/emicklei/go-restful                    v2.5.0            v2.5.0            2dd4403   2dd4403  2
github.com/emicklei/go-restful-swagger12          1.0.1             1.0.1             dcef7f5   dcef7f5  1
github.com/fsnotify/fsnotify                      v1.4.7            v1.4.7            c282820   c282820  1
github.com/ghodss/yaml                            v1.0.0            v1.0.0            0ca9ea5   0ca9ea5  1
github.com/giantswarm/apiextensions               branch master     branch master     f0b8bcd   126d661  6
github.com/giantswarm/certs                       branch master     branch master     77bba48   cb5abd7  2
github.com/giantswarm/k8scloudconfig              branch master     branch master     789bebc   2512942  2
github.com/giantswarm/k8shealthz                  branch master     branch master     5eaeffb   5eaeffb  1
github.com/giantswarm/microendpoint               branch master     branch master     7d8dc82   7d8dc82  4
github.com/giantswarm/microerror                  branch master     branch master     325932f   325932f  1
github.com/giantswarm/microkit                    branch master     branch master     489a866   d48ce8e  15
github.com/giantswarm/micrologger                 branch master     branch master     216e919   216e919  3
github.com/giantswarm/microstorage                branch master     branch master     4411a63   4411a63  2
github.com/giantswarm/operatorkit                 branch master     branch master     9447814   59017b0  10
github.com/giantswarm/randomkeys                  branch master     branch master     156c1ca   156c1ca  2
github.com/giantswarm/versionbundle               branch master     branch master     5d9ca79   9172db5  1
github.com/go-kit/kit                             v0.6.0            v0.6.0            4dc7be5   4dc7be5  3
github.com/go-logfmt/logfmt                       v0.3.0            v0.3.0            390ab79   390ab79  1
github.com/go-openapi/jsonpointer                 branch master     branch master     779f453   779f453  1
github.com/go-openapi/jsonreference               branch master     branch master     36d33bf   36d33bf  1
github.com/go-openapi/spec                        branch master     branch master     8ed76de   f3499b5  1
github.com/go-openapi/swag                        branch master     branch master     84f4bee   84f4bee  1
github.com/go-resty/resty                         v1.2              v1.2              00cb083   00cb083  1
github.com/go-stack/stack                         v1.7.0            v1.7.0            259ab82   259ab82  1
github.com/gogo/protobuf                          v0.5              v0.5              342cbe0   342cbe0  2
github.com/golang/glog                            branch master     branch master     23def4e   23def4e  1
github.com/golang/protobuf                        branch master     branch master     9255415   bbd03ef  5
github.com/google/btree                           branch master     branch master     e89373f   e89373f  1
github.com/google/gofuzz                          branch master     branch master     24818f7   24818f7  1
github.com/googleapis/gnostic                     v0.1.0            v0.1.0            ee43cbb   ee43cbb  3
github.com/gorilla/context                        v1.1              v1.1              1ea2538   1ea2538  1
github.com/gorilla/mux                            v1.6.1            v1.6.1            53c1911   53c1911  1
github.com/gregjones/httpcache                    branch master     branch master     2bcd89a   2bcd89a  2
github.com/hashicorp/golang-lru                   branch master     branch master     0a025b7   0fb14ef  2
github.com/hashicorp/hcl                          branch master     branch master     23c074d   23c074d  9
github.com/inconshreveable/mousetrap              v1.0              v1.0              76626ae   76626ae  1
github.com/json-iterator/go                       1.0.5             1.0.5             28452fc   28452fc  1
github.com/juju/errgo                             branch master     branch master     08cceb5   08cceb5  1
github.com/juju/ratelimit                         1.0.1             1.0.1             59fac50   59fac50  1
github.com/kr/logfmt                              branch master     branch master     b84e30a   b84e30a  1
github.com/magiconair/properties                  v1.7.4            v1.7.4            d419a98   d419a98  1
github.com/mailru/easyjson                        branch master     branch master     32fa128   32fa128  3
github.com/matttproud/golang_protobuf_extensions  v1.0.0            v1.0.0            3247c84   3247c84  1
github.com/mitchellh/mapstructure                 branch master     branch master     b4575ee   a4e142e  1
github.com/pelletier/go-toml                      v1.1.0            v1.1.0            acdc450   acdc450  1
github.com/petar/GoLLRB                           branch master     branch master     53be0d3   53be0d3  1
github.com/peterbourgon/diskv                     v2.0.1            v2.0.1            5f041e8   5f041e8  1
github.com/prometheus/client_golang               branch master     branch master     06bc6e0   9bb6ab9  2
github.com/prometheus/client_model                branch master     branch master     99fa1f4   99fa1f4  1
github.com/prometheus/common                      branch master     branch master     89604d1   89604d1  3
github.com/prometheus/procfs                      branch master     branch master     cb41470   cb41470  4
github.com/spf13/afero                            v1.0.2            v1.0.2            bb8f192   bb8f192  2
github.com/spf13/cast                             v1.1.0            v1.1.0            acbeb36   acbeb36  1
github.com/spf13/cobra                            v0.0.1            v0.0.1            7b2c5ac   7b2c5ac  1
github.com/spf13/jwalterweatherman                branch master     branch master     7c0cea3   7c0cea3  1
github.com/spf13/pflag                            v1.0.0            v1.0.0            e57e3ee   e57e3ee  1
github.com/spf13/viper                            v1.0.0            v1.0.0            25b30aa   25b30aa  1
github.com/tylerb/graceful                        v1.2.15           v1.2.15           4654dfb   4654dfb  1
golang.org/x/net                                  branch master     branch master     0ed95ab   2fb46b1  6
golang.org/x/sync                                 branch master     branch master     fd80eb9   fd80eb9  1
golang.org/x/sys                                  branch master     branch master     3dbebcf   37707fd  1
golang.org/x/text                                 branch master     branch master     e19ae14   e19ae14  15
gopkg.in/inf.v0                                   v0.9.0            v0.9.0            3887ee9   3887ee9  1
gopkg.in/yaml.v2                                  branch v2         branch v2         d670f94   d670f94  1
k8s.io/api                                        kubernetes-1.8.5  kubernetes-1.8.5  9b9dca2            24
k8s.io/apiextensions-apiserver                    kubernetes-1.8.5  kubernetes-1.8.4  996a70a            5
k8s.io/apimachinery                               kubernetes-1.8.5  kubernetes-1.8.3  5134afd            40
k8s.io/client-go                                  kubernetes-1.8.5  kubernetes-1.8.5  1b75876            66
k8s.io/kube-openapi                               branch master     branch master     a07b7bb   275e2ce  1
```

Reply:
At least makes sense.

Reply:
In context of cluster-service we said we want to pin everything down. KVM does not have this yet because it was one of the first we moved to dep.